packages/bootstrap/scss/stepper/_variables.scss

```diff
@@ -82,6 +82,11 @@ $kendo-stepper-progressbar-text: null !default;
 $kendo-stepper-progressbar-fill-bg: null !default;
 $kendo-stepper-progressbar-fill-text: null !default;
 
+/// @deprecated Use `$kendo-stepper-content-transition` instead.
 $kendo-stepper-content-transition-property: height !default;
+/// @deprecated Use `$kendo-stepper-content-transition` instead.
```

Comment:
Use var(--kendo-motion-medium-2, .3s) !default; (like in the Default theme)

Reply:
Since `$kendo-stepper-content-transition` uses `k-motion` this is not needed. I'll remove it from the Default theme.